app/src/main/res/layout/fragment_user_profile.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".fragments.UserProfileFragment">
+
+
+    <TextView
+        android:id="@+id/settings"
+        android:layout_width="126dp"
```

Comment:
Since the text size is already defined and you're using a horizontal bias, `wrap_content` may be preferable than a fixed size